shell/rapi2/Makefile

```diff
@@ -0,0 +1,91 @@
+LOCAL_PATH := $(call my-dir)
+FOR_LINUX :=1
+WEBUI :=1
+#NO_REND := 1
+#NO_X11 := 1
+RZDCY_SRC_DIR = ../../core
+
+include $(RZDCY_SRC_DIR)/core.mk
+
+#CC_PREFIX ?= arm-none-linux-gnueabi-
+
+CXX=${CC_PREFIX}g++
+CC=${CC_PREFIX}gcc
+AS=${CC_PREFIX}as
+STRIP=${CC_PREFIX}strip
+
+LD=${CC}
+
+MFLAGS := -marm -march=armv7-a -mtune=cortex-a9 -mfpu=neon -mfloat-abi=hard -funroll-loops
+ASFLAGS := -march=armv7-a -mfpu=neon -mfloat-abi=softfp
+
+LDFLAGS	:=	-g -Wl,-Map,$(notdir $@).map,--gc-sections -Wl,-O3 -Wl,--sort-common 
+
+
+CXXFLAGS := -g -O3  -D RELEASE -c -D TARGET_BEAGLE -D TARGET_LINUX_ARMELv7  
+CXXFLAGS += -frename-registers -fno-strict-aliasing -fsingle-precision-constant 
+CXXFLAGS += -ffast-math -ftree-vectorize -fprefetch-loop-arrays 
+#-std=c++0x
+
+CXXFLAGS += $(CFLAGS) $(MFLAGS) -fno-exceptions -fno-rtti
+CXXFLAGS +=  -DARM_HARDFP -DGLES
+
+
+ifdef PGO_MAKE
+	CXXFLAGS += -fprofile-generate -pg
+	LDFLAGS	+= -fprofile-generate
+else
+	CXXFLAGS += -fomit-frame-pointer
+endif
+
+ifdef PGO_USE
+	CXXFLAGS += -fprofile-use
+endif
+
+
+ifdef LTO_TEST
+	CXXFLAGS += -flto -fwhole-program 
+	LDFLAGS +=-flto -fwhole-program 
+endif
+
+INCS	:= -I/opt/vc/include/ -I/opt/vc/include/interface/vmcs_host/linux -I/opt/vc/include/interface/vcos/pthreads  -I$(RZDCY_SRC_DIR) -I$(RZDCY_SRC_DIR)/deps -I$(RZDCY_SRC_DIR)/khronos -I../linux-deps/include
+
+LIBS    :=  -L/opt/vc/lib/  -L../linux-deps/lib
+LIBS    += -lm -lrt -lEGL -lGLESv2 #-lglslcompiler -lIMGegl -lpvr2d -lsrv_um 
+LIBS    += -lpthread -lasound -lX11 -lXdmcp -lXau  -lbcm_host
```

Comment:
Not sure, but that was done by @skmp he basicly just copied the Beagle Board Makefile and adapted it.
But yes I think it's not needed.